data/management/commands/fetch_external_data.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os.path
+
+from django.core.management.base import BaseCommand
+import requests
+
+from community.git import get_org_name
+
+
+class Command(BaseCommand):
+    help = 'Fetch external data'
+
+    def add_arguments(self, parser):
+        parser.add_argument('output_dir', nargs='?', type=str)
+        parser.add_argument('repo_name', nargs='?', type=str)
+        parser.add_argument('filenames', nargs='+', type=str)
+        parser.add_argument('--allow_failure', action='store_true',
+                            default=False, help='Don\'t raise exceptions.')
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger(__name__)
+        output_dir = options.get('output_dir')
+        repo_name = options.get('repo_name')
+        filenames = options.get('filenames')
+        allow_failure = options.get('allow_failure')
+        org_name = get_org_name()
+
+        for filename in filenames:
+            url = ('http://' + org_name + '.github.io/'
+                   + repo_name + '/' + filename)
+            logger.info('Get external data from %s' % url)
+            try:
+                r = requests.get(url, timeout=10)
+                r.raise_for_status()
+            except Exception as ex:
+                logger.warning('Get external data from %s failed: %s '
+                               'Try backup url' % (url, ex))
+                url = ('https://' + org_name + '-'
```

Comment:
again, please build this in `community.git`.
already requested once at https://github.com/coala/community/pull/143#discussion_r194249685

netlify is just another potential host.

We also need `community.git` to support 'gitlab pages'.
(e.g. for orgs which are GitLab only)
And it would be great if orgs can choose which hoster they are using, and switch between then.  e.g. my real job, NGO https://ciptamedia.org/ is @cipta-media and https://gitlab.com/ciptamedia and also on Netlify , and it is built so that it could be deployed on any of them, so that changes of Terms of Use or a hoster charging for the service are not a gun-at-our-head; instead it is a slightly annoying retest and DNS update to discard a host which is no longer suitable. (I guess we could also add BitBucket for extra redundancy, and possibly others .. ;-)

I'm still not seeing the benefit of allowing two potential fetch locations for the same file, and `allow_failure`.
This is hiding failure, and the assumption that the backup is OK is not very sane IMO.  Likely the primary gets broken, and nobody fixes it because there is a backup (or a PR gets ignored in the queue), and then the backup breaks, and then the primary gets fixed but the backup stays broken, and then ... ;-)

The problems with the data not being retained in one of those hosters is a problem with the build system of gh-board, and one of the solutions is incredibly easy : https://github.com/coala/gh-board/issues/52  mentioned previously at https://github.com/coala/community/pull/143#discussion_r194706920 and at https://github.com/coala/community/pull/143#discussion_r194249921

If we remove this dual host thing, we could easily add the `fetch_external_data.py` functionality into the `fetch_deployed_data.py` command by adding an optional 'repo' argument.

failure is our friend.  exceptions with full backtrace are useful to get operational quickly.